srcpkgs/planner/update

```diff
@@ -0,0 +1 @@
+ignore="6.2.3"
```

Comment:
please also add a comment why this version is ignored - it was tagged by mistake

Reply:
Done.